app/models/host.rb

```diff
@@ -332,74 +330,56 @@ def start
   end
 
   def stop
-    if supports?(:stop)
+    if verbose_supports?(:stop)
       check_policy_prevent("request_host_stop", "ipmi_power_off")
-    else
-      _log.warn("Cannot stop because <#{unsupported_reason(:stop)}>")
     end
   end
 
   def standby
-    if supports?(:standby)
+    if verbose_supports?(:standby)
       check_policy_prevent("request_host_standby", "vim_power_down_to_standby")
-    else
-      _log.warn("Cannot go into standby mode because <#{unsupported_reason(:standby)}>")
     end
   end
 
   def enter_maint_mode
-    if supports?(:enter_maint_mode)
+    if verbose_supports?(:enter_maint_mode)
       check_policy_prevent("request_host_enter_maintenance_mode", "vim_enter_maintenance_mode")
-    else
-      _log.warn("Cannot enter maintenance mode because <#{unsupported_reason(:enter_maint_mode)}>")
     end
   end
 
   def exit_maint_mode
-    if supports?(:exit_maint_mode)
+    if verbose_supports?(:exit_maint_mode)
       check_policy_prevent("request_host_exit_maintenance_mode", "vim_exit_maintenance_mode")
-    else
-      _log.warn("Cannot exit maintenance mode because <#{unsupported_reason(:exit_maint_mode)}>")
     end
   end
 
   def shutdown
-    if supports?(:shutdown)
+    if verbose_supports?(:shutdown)
       check_policy_prevent("request_host_shutdown", "vim_shutdown")
-    else
-      _log.warn("Cannot shutdown because <#{unsupported_reason(:shutdown)}>")
     end
   end
 
   def reboot
-    if supports?(:reboot)
+    if verbose_supports?(:reboot)
       check_policy_prevent("request_host_reboot", "vim_reboot")
-    else
-      _log.warn("Cannot reboot because <#{unsupported_reason(:reboot)}>")
     end
   end
 
   def enable_vmotion
-    if supports?(:enable_vmotion)
+    if verbose_supports?(:enable_vmotion)
       check_policy_prevent("request_host_enable_vmotion", "vim_enable_vmotion")
-    else
-      _log.warn("Cannot enable vmotion because <#{unsupported_reason(:enable_vmotion)}>")
     end
   end
 
   def disable_vmotion
-    if supports?(:disable_vmotion)
+    if verbose_supports?(:disable_vmotion)
       check_policy_prevent("request_host_disable_vmotion", "vim_disable_vmotion")
-    else
-      _log.warn("Cannot disable vmotion because <#{unsupported_reason(:disable_vmotion)}>")
     end
   end
 
   def vmotion_enabled?
-    if supports?(:vmotion_enabled)
+    if verbose_supports?(:vmotion_enabled)
```

Comment:
This will log `Cannot Vmotion enabled because ...` maybe take an optional parameter to override the `#{feature.to_s.humanize}`?